src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -328,7 +334,7 @@ private void createStudyAsync(File caseFile, String originalFilename, String use
                 String caseFormat = getCaseFormat(caseUuid, basicStudyInfos.getId(), userId);
                 NetworkInfos networkInfos = persistentStore(caseUuid, basicStudyInfos.getId(), userId, importReportUuid, null);
                 LoadFlowParameters loadFlowParameters = LoadFlowParameters.load();
-                insertStudy(basicStudyInfos.getId(), userId, networkInfos, caseFormat, caseUuid, false, toEntity(loadFlowParameters), importReportUuid);
+                insertStudy(basicStudyInfos.getId(), userId, networkInfos, caseFormat, caseUuid, false, originalFilename, toEntity(loadFlowParameters), importReportUuid);
```

Comment:
Why not using getCaseInfos instead of getCaseFormat, like in the previous createStudyAsync method ?

Reply:
getCaseInfos retrieves both the case format and its name.
In this use case we already have the case name in hand since it's being uploaded alongside the study creation so getCaseFormat seemed more appropriate to me.